pom.xml

```diff
@@ -192,6 +192,14 @@
             <version>2.7.4</version>
             <scope>provided</scope>
         </dependency>
+
+        <!-- will be included and relocated by shade plugin -->
+        <dependency>
+            <groupId>org.owasp.encoder</groupId>
+            <artifactId>encoder</artifactId>
+            <version>1.2.1</version>
+            <scope>compile</scope>
```

Comment:
Using `compile` does not make that dependency transitive? Or shade will remove this dependency from the pom.ml